std/algorithm/sorting.d

```diff
@@ -294,7 +296,9 @@ if (isForwardRange!Range)
 
 /**
 Like `isSorted`, returns `true` if the given `values` are ordered
-according to the comparison operation `less`. Unlike `isSorted`, takes values
+according to the comparison operation `less`.
+
+Unlike `isSorted`, takes values
 directly instead of structured in a range.
```

Comment:
This is the one change I don't consider an improvement. The rest of the changes look good to me.

Reply:
Yes, I agree that it is fine to have both on the same paragraph.

Reply:
I changed it back